components/Article/ActionBarOverlay.js

```diff
@@ -7,21 +7,12 @@ const ACTIONBAR_FADE_AREA = 400
 const ActionBarOverlay = ({ children, audioPlayerVisible, inNativeApp }) => {
   const [colorScheme] = useColorContext()
   const [overlayVisible, setOverlayVisible] = useState(false)
-  const [bottomPosition, setBottomPosition] = useState()
   const lastY = useRef()
   const diff = useRef(0)
 
-  useEffect(() => {
```

Comment:
I think we still need this for mobile browsers (non-native-app).

Reply:
in browser it's always 44 anyhow – doesn't make a difference

Reply:
true!